### ISSUE TITLE
Changing prefix will change adjustment value when resize-pane with shortcut

### ISSUE THREAD

**Issue by c02y** [tmux/tmux#1066]
- CentOS 6.5 32-bit
- tmux master
- TERM: xterm-256color

Changing default prefix key which is `C-b` by default to another such as `M-c` or `M-s` or `M-d` (I just tried three random keys), the default adjustment value which is 1 line by default when resize-pane with shortcut such as `prefix+C-UP` will be changed to 5. This is really weird.

The command line to resize pane such as `tmux resize-pane -U` or `tmux resize-pane -U 1` works as expected. If I use the default prefix key which is `C-b` to resize-pane with shortcut such as `C-b C-UP`, it works as expected too.

It seems this bug exists for a long time (I noticed it long time ago), I found it annoying recently since it resize more lines than I want, and I tried to search for a solution to set the value of adjustment but it seems tmux provides no option of adjustment value, and then after I tested my .tmux.conf piece by piece and found that it is the problem `set -g prefix-key ...`.

**Comment by nicm** [comment]
I think you are pressing C-b Up when your prefix is C-b but M-c M-Up when
it is M-c.

Try unbinding the M-Up, M-Down, M-Left, M-Down resize key bindings and see
if the problem goes away.



On 11 Sep 2017 9:32 am, "c02y" <notifications@github.com> wrote:


   - CentOS 6.5 32-bit
   - tmux master
   - TERM: xterm-256color

Changing default prefix key which is C-b by default to another such as M-c
or M-s or M-d (I just tried three random keys), the default adjustment
value which is 1 line by default when resize-pane with shortcut such as
prefix+C-UP will be changed to 5. This is really weird.

The command line to resize pane such as tmux resize-pane -U or tmux
resize-pane -U 1 works as expected. If I use the default prefix key which
is C-b to resize-pane with shortcut such as C-b C-UP, it works as expected
too.

It seems this bug exists for a long time (I noticed it long time ago), I
found it annoying recently since it resize more lines than I want, and I
tried to search for a solution to set the value of adjustment but it seems
tmux provides no option of adjustment value, and then after I tested my
.tmux.conf piece by piece and found that it is the problem set -g
prefix-key ....

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/tmux/tmux/issues/1066>, or mute the thread
